documentation/book/api/paths.adoc

```diff
@@ -1018,7 +1018,7 @@ Sends one or more records to a given topic, optionally specifying a partition, k
 |===
 |Type|Name|Description|Schema
 |**Path**|**topicname** +
-__required__|Name of the topic which you want to either retrieve metadata for, or to send records to.|string
```

Comment:
Not sure we need the 'which you want to' in any of these descriptions. 
What about _Name of the topic to to send records to or retrieve metadata from_?
Same for the partition descriptions too.

Reply:
That sounds good. I think we should avoid the use of 'you' in those short description phrases. @ppatierno How do you think?